exporter/exporter.go

```diff
@@ -193,7 +193,7 @@ func NewKvrocksExporter(kvrocksURI string, opts Options) (*Exporter, error) {
 		"commands_duration_seconds_bucket":     {txt: `Histogram of the amount of time in seconds spent per command`, lbls: []string{"cmd"}},
 		"commands_duration_seconds_total":      {txt: `Total amount of time in seconds spent per command`, lbls: []string{"cmd"}},
 		"commands_total":                       {txt: `Total number of calls per command`, lbls: []string{"cmd"}},
-		"connected_slave_lag_seconds":          {txt: "Lag of connected slave", lbls: []string{"slave_ip", "slave_port", "slave_state"}},
```

Comment:
This change will cause a compatibility issue for other users. It should be good to keep the same name with Redis but has different meaning.